docs/theming/component-variables.mdx

```diff
@@ -158,6 +158,23 @@ Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/style
 
 Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/styles/Thread/Thread-theme.scss](https://github.com/GetStream/stream-chat-css/blob/main/src/v2/styles/Thread/Thread-theme.scss)
 
+## Typing indicator
+
+| Name                                                | Description                                                   |
+| --------------------------------------------------- | ------------------------------------------------------------- |
+| `--str-chat__typing-indicator-font-family`          | The font used in the component                                |
+| `--str-chat__typing-indicator-color`                | The text/icon color of the component                          |
+| `--str-chat__typing-indicator-background-color`     | The background color of the component                         |
+| `--str-chat__typing-indicator-border-radius`        | The border radius used for the borders of the component       |
+| `--str-chat__typing-indicator-border-block-start`   | Top border of the component                                   |
+| `--str-chat__typing-indicator-border-block-end`     | Bottom border of the component                                |
+| `--str-chat__typing-indicator-border-inline-start`  | Left (right in RTL layout) border of the component            |
+| `--str-chat__typing-indicator-border-inline-start`  | Right (lrft in RTL layout) border of the component            |
```

Comment:
```suggestion
| `--str-chat__typing-indicator-border-inline-end`  | Right (left in RTL layout) border of the component            |
```

Reply:
Will fix this after the ongoing PRs merged into theming-v2